apps/trading/pages/portfolio/withdraw/index.page.tsx

```diff
@@ -0,0 +1,35 @@
+import { Web3Container } from '../../../components/web3-container';
+import { useRouter } from 'next/router';
+import { useMemo } from 'react';
+import { WithdrawPageContainer } from './withdraw-page-container';
+import { t } from '@vegaprotocol/react-helpers';
+
+const Withdraw = () => {
+  const { query } = useRouter();
+
+  // AssetId can be specified in the query string to allow link to deposit a particular asset
+  const assetId = useMemo(() => {
+    if (query.assetId && Array.isArray(query.assetId)) {
+      return undefined;
+    }
+
+    if (Array.isArray(query.assetId)) {
+      return undefined;
+    }
+
+    return query.assetId;
+  }, [query]);
+
+  return (
+    <Web3Container>
```

Comment:
chore (non-blocking): change to render function instead of children as per the discussion on the Order Book PR?